### ISSUE TITLE
Separate branding for subsites

### ISSUE THREAD

**Issue by camfindlay** [adrexia/silverstripe-brand#1]
Coz we know how everyone loves subsites...

So far in a 3.3.x core install you cannot see the "Brand" item in the CMS nav. Editing anything in the Brand section applies it to all subsites.

Will need some sort of "if Subsites installed" flag so we don't need to add extra tables and relations if a user isn't making use of that module.

Any thoughts? I'm happy to help code something when I get the time (using this for a community project :)).

**Comment by adrexia** [comment]
This is largely to do with the reliance on Little-Giant/silverstripe-singleobjectadmin. I've opened a pull request there to get the fix needed (https://github.com/Little-Giant/silverstripe-singleobjectadmin/pull/6/commits/be44b898d4e0944b19539cef28c7639e0d456910). Then, it seems to be a matter of using an augmentSQL function (at least without making singlemodeladmin subsite aware). 

I've added what's needed into this module  - https://github.com/adrexia/silverstripe-subsite-modeladmins - and tested it with the singlemodeladmin fix. Seems to work. :)

**Comment by camfindlay** [comment]
nice thanks @adrexia :) Let me know when you tag a 0.1.0 release ;) I'll see what I can add.

**Comment by adrexia** [comment]
I've added subsite docs and the pull request has been merged. :) 

https://github.com/adrexia/silverstripe-brand#subsite-support

Not sure when it'll be ready for a release - I only conceived of and started this on Friday. It's likely to change a fair bit as I integrate it, but that's more likely to be in the way of additional functionality I think. The 'theme' I'm working on with this is specifically the sort I want to be able to set up for people over and over again without too much custom dev work, so there's likely to be other challenges there.
